### PR TITLE
Testnet prod sync

### PR DESCRIPTION
**Pull request by just-a-node**
## Type of change

- [ ] Docs change / dependency upgrade
- [ ] Configuration / tooling changes
- [ ] Refactoring
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires changes in customer code

## Related Issue(s)

Fixes

### CODE REVIEW

File: packages/examples/sdk-server/package.json [hunk 1]
```diff
@@ -21,6 +21,7 @@
     "@connext/nxtp-txservice": "workspace:*",
     "@connext/nxtp-utils": "workspace:*",
     "@connext/sdk": "workspace:*",
+    "@fastify/redis": "^6.1.1",
```
Comment (LayneHaber) [inline packages/examples/sdk-server/package.json:24]:
```suggestion
    "@fastify/redis": "6.1.1",
```

File: packages/examples/sdk-server/src/index.ts [hunk 2]
```diff
@@ -14,24 +15,26 @@ export const sdkServer = async (): Promise<FastifyInstance> => {
...
     console.info("No NXTP_CONFIG exists, using config file and individual env vars");
   }
-  try {
-    let json: string;
-    const path = process.env.NXTP_CONFIG_FILE ?? "config.json";
-    if (fs.existsSync(path)) {
-      json = fs.readFileSync(path, { encoding: "utf-8" });
-      configJson = JSON.parse(json);
+
+  if (Object.keys(configJson).length === 0) {
```
Comment (LayneHaber) [inline packages/examples/sdk-server/src/index.ts:26]: do we ever partial config from the `env.NXTP_CONFIG` and a file?

Comment (LayneHaber) [reply packages/examples/sdk-server/src/index.ts:26]: we should either update this to error explicitly if both non-empty config and json file are surprised, or combine the configs (probably explicitly error)

File: packages/examples/sdk-server/src/index.ts [hunk 3]
```diff
@@ -75,12 +85,12 @@ export const sdkServer = async (): Promise<FastifyInstance> => {
...
   });
 
-  server.register(baseRoutes, sdkBase);
+  server.register(baseRoutes, { sdkBaseInstance: sdkBase, cacheConfig: configJson.cache });
   server.register(poolRoutes, sdkPool);
   server.register(utilsRoutes, sdkUtils);
   server.register(routerRoutes, sdkRouter);
 
-  server.listen(8080, (err, address) => {
+  server.listen({ port: 8080 }, (err, address) => {
```
Comment (LayneHaber) [inline packages/examples/sdk-server/src/index.ts:93]: the port should be config-provided (in accordance with our other server configs)

**Review by LayneHaber — CHANGES_REQUESTED** [review]

**Review by preethamr — APPROVED** [review]

**Comment by just-a-node** [comment]
Closing to resolve comments in https://github.com/connext/monorepo/pull/4418

**Pull request closed without merging**